docs/.vitepress/theme/index.js

```diff
@@ -7,19 +8,34 @@ import ComponentQuestions from '../../components/ComponentQuestions.md';
 import ComponentsStatus from '../ComponentsStatus.vue';
 import Do from '../Do.vue';
 import DoDont from '../DoDont.vue';
+import QrTable from '../qr-table.vue';
+import QrColorTable from '../qr-color-table.vue';
+import WidthController from '../WidthController.vue';
+import { IconStarFull32 } from '@warp-ds/icons/vue';
 import '../bootExamples.js';
 import './custom.css';
+import warpThemeSwitcher from "../warp-theme-switcher.js";
 
 export default {
   ...DefaultTheme,
-  enhanceApp({ app }) {
-    app.component('ApiTable', ApiTable);
-    app.component('ThemeSwitcher', ThemeSwitcher);
-    app.component('TabsContent', TabsContent);
-    app.component('ComponentDesignGuidelines', ComponentDesignGuidelines);
-    app.component('ComponentQuestions', ComponentQuestions);
-    app.component('ComponentsStatus', ComponentsStatus);
-    app.component('Do', Do);
-    app.component('DoDont', DoDont);
+  Layout() {
+    return h(DefaultTheme.Layout, null, {
+      'nav-bar-content-before': () => h(ThemeSwitcher)
```

Comment:
Would it make sense to have the theme switcher appear after the search field? I have a feeling it's not that relevant to most users what theme is applied. 
Also, when the select element is in focus, it gets a red outline, while other interactive elements on that page have blue outline color.
Finally, a small thing, but could we add some horizontal padding to the content div so the switcher is a bit further to the right.
![Screenshot 2024-06-10 at 14 12 18](https://github.com/warp-ds/tech-docs/assets/41303231/f3c7f6cc-4199-4df9-a880-859c1e6e85ba)

Reply:
Pushed tweaks to the theme switcher: 
- Moved it to the right of the search field
- Added it to the mobile screen menu
- Minor layout fixes etc